app/controllers/users/mfa_selection_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Users
+  class MfaSelectionController < ApplicationController
+    include UserAuthenticator
+    include MfaSetupConcern
+
+    before_action :authenticate_user
+    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :multiple_factors_enabled?
+
+    def index
+      @two_factor_options_form = TwoFactorOptionsForm.new(current_user)
+      @presenter = two_factor_options_presenter
+      analytics.track_event(Analytics::USER_REGISTRATION_2FA_ADDITIONAL_SETUP_VISIT)
+    end
+
+    def update
+      result = submit_form
+      analytics.track_event(Analytics::USER_REGISTRATION_2FA_ADDITIONAL_SETUP, result.to_h)
```

Comment:
ditto, new style would look like:
```suggestion
      analytics.user_registration_2fa_additional_setup(**result.to_h)
```